config/teleop.yaml

```diff
@@ -4,20 +4,28 @@ teleop:
   ra_controls:
     joint_a:
       multiplier: 1
+      slow_mode_multiplier: 0.5
```

Comment:
selfish request but keep a,c,e,f, and finger at 0.5/-0.5 and keep the rest at 1 lol

Reply:
Agree about joint B but... joint D? It's not super fast but it's not slow

Reply:
ok maybe keep joint d... i forgot that we swapped to brushless so it's much faster now lol. i was kinda basing it on code from last year